Cannot make clientaway

ZNC version:
znc --version
ZNC - 1.5.0-nightly-20150205-4d17d621 - http://znc.in
IPv6: yes, SSL: yes, DNS: threads, charset: yes

Steps followed:
root@server1 <~/znc-git/znc-git-2015-02-05> git clone https://github.com/kylef/znc-contrib.git 
remote: Counting objects: 79, done.
remote: Total 79 (delta 0), reused 0 (delta 0)
Unpacking objects: 100% (79/79), done.
root@server1 <~/znc-git/znc-git-2015-02-05> cd znc-contrib
root@server1 (master) <~/znc-git/znc-git-2015-02-05/znc-contrib> make clientaway
Building "clientaway.so" for ZNC 1.5.0... clientaway.cpp: In member function ‘virtual CModule::EModRet CClientAwayMod::OnUserRaw(CString&)’:
clientaway.cpp:152:18: error: ‘class CIRCNetwork’ has no member named ‘ClearQueryBuffer’
[ !! ] Error while building "clientaway.so"
make: *** [clientaway] Error 1

Thanks!

Same issue here

```
../../modules/clientaway.cpp: In member function ‘virtual CModule::EModRet CClientAwayMod::OnUserRaw(CString&)’:
../../modules/clientaway.cpp:152:18: error: ‘class CIRCNetwork’ has no member named ‘ClearQueryBuffer’
```

> ZNC - 1.7.0-git-23-e116d9e - http://znc.in
> IPv6: yes, SSL: yes, DNS: threads, charset: yes

It appears that this module is incompatible with 1.6.0. @kylef also said that they are running 1.1 or something.

--- a/clientaway.cpp
+++ b/clientaway.cpp
@@ -10,9 +10,7 @@
 #include <znc/User.h>
 #include <znc/IRCNetwork.h>
 #include <znc/Chan.h>
-#if (VERSION_MAJOR >= 1) && (VERSION_MINOR >= 5)
-#include <znc/Query.h>
-#endif
+
 using std::vector;
 
 #define AWAY_DEFAULT_REASON "Auto away at %time%"
@@ -151,13 +149,7 @@ public:
 						}
 					}
 
-#if (VERSION_MAJOR >= 1) && (VERSION_MINOR >= 5)
-					for (CQuery* pQuery : m_pNetwork->GetQueries()) {
-						m_pNetwork->DelQuery(pQuery->GetName());
-					}
-#else
 					m_pNetwork->ClearQueryBuffer();
-#endif
 
 					if (GetAutoAway() && m_pNetwork->IsIRCAway()) {
 						PutIRC("AWAY");

